tests/test_sensors.py

```diff
@@ -0,0 +1,45 @@
+import habitat_sim.bindings as hsim
+import habitat_sim
+import habitat_sim.utils
+import habitat_sim.errors
+import numpy as np
+import quaternion
+import pytest
+import os.path as osp
+
+
+_test_scene = osp.abspath(
+    osp.join(osp.dirname(__file__), "17DRP5sb8fy", "17DRP5sb8fy.glb")
```

Comment:
@erikwijmans, really like sensor_tests. We have public testing scenes (http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip). Can that be used for testing?

Reply:
Those don't have semantic labels unfortunately.  Although it is probably worth adding tests for RGB and Depth on public testing scenes

Reply:
Added.  That was easier than I thought it would be.  Yay pytest!